pkgs/development/libraries/libutp/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake}:
+
+stdenv.mkDerivation rec {
+  pname = "libutp";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "transmission";
+    repo = "${pname}";
+    rev = "fda9f4b3db97ccb243fcbed2ce280eb4135d705b";
```

Comment:
https://github.com/bittorrent/libutp/issues/129

Also I think we want to fetch the latest commit here 2b364cbb0650bdab64a5de2abb4518f9f228ec44.